app/src/db/migrations/20220130133615_000-init.js

```diff
@@ -0,0 +1,129 @@
+const stamps = require('../stamps');
+const uuid = require('uuid');
+
+const CREATED_BY = uuid.NIL;
+
+exports.up = function (knex) {
+  return Promise.resolve()
+
+    // Create user table and add a dummy system user (to link migration CreatedBy stamps)
+    .then(() => knex.schema.createTable('oidc_user', table => {
+      table.string('oidcId').primary();
+      table.string('firstName');
+      table.string('fullName');
+      table.string('lastName');
+      table.string('username').notNullable().index();
+      table.string('email').index();
+      table.boolean('active').notNullable().defaultTo(true);
+      stamps(knex, table);
+    }))
+    .then(() => {
+      const items = [
+        {
+          oidcId: CREATED_BY,
+          username: 'System',
+          active: false
+        },
+      ];
+      return knex('oidc_user').insert(items);
+    })
+    .then(() => knex.schema.createTable('identity_provider', table => {
+      table.string('code').primary();
+      table.string('display').notNullable();
+      table.string('idpAlias');
```

Comment:
kind of wondering if we need this field. i guess it can't hurt and could be useful

Reply:
Probably not for us, but would support the ability to have an alias be different accross envs (like happened to us on CHEFS).
So decouples the lookup code (primary key) with the changeable alias in the provider, but likely always the same.